src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -214,8 +217,8 @@ def resize_subvolume(self, **kwargs):
                             [{'bytes_used': usedbytes},{'bytes_quota': nsize},
                              {'bytes_pcent': "undefined" if nsize == 0 else '{0:.2f}'.format((float(usedbytes) / nsize) * 100.0)}],
                             indent=4, sort_keys=True), ""
-        except VolumeException as ve:
-            ret = self.volume_exception_to_retval(ve)
+        except (CephfsConnectionException, VolumeException) as e:
```

Comment:
pybind/mgr/volumes/fs/operations/volume.py:open_volume{_lockless} can raise VolumeException which can be converted to mgr return value here.

Reply:
Done